elastic/beats
# beat/kibana module - add {module}.elasticsearch.cluster.id property

### Summary
This change adds the `beat.elasticsearch.cluster.id` property to both `state` and `stats` beat's metricsets, which is required for the SM UI to retrieve beats that outputs their events to a selected cluster.
It also reverts [this change](https://github.com/elastic/beats/commit/c428cb17ff0aa6613cf065b2779481a8451e1658) as we should not be writing to other root module properties from a given module. This requires [alias changes in elasticsearch](https://github.com/elastic/elasticsearch/pull/82029) to be functional.

The cluster ID was already available in the `state` metricset so it was just a matter of adding that property to the right path. For the `stats` metricset I reused the [7.16 xpack logic](https://github.com/elastic/beats/blob/7.16/metricbeat/module/beat/stats/data_xpack.go#L80).

### Testing
- start one metricbeat with `elasticsearch-xpack` module enabled
- start another metricbeat that monitors the first one
- verify relevant beats information appear in the SM UI
